### DISCUSSION TITLE
PKGi PS3 v1.2.0

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/pkgi-ps3#80] [General]
### Added

* Internationalization support
* Language detection based on PS3 settings
* Spanish translation
* Online scan for available content updates
* Support for TLS v1.2 using newer libcurl+polarSSL

<hr /><em>This discussion was created from the release <a href='https://github.com/bucanero/pkgi-ps3/releases/tag/v1.2.0'>PKGi PS3 v1.2.0</a>.</em>

**Comment by Fabianvcp** [comment]
error no file(s) pkg1.txt

**Comment by AllenWalker1707** [comment]
Aucun fichier pkg pour le PKGI PS3 version 1.2.0 donc inutilisable pour le moment.
Veuillez le mettre a jour pour que tout le monde puisse l'utiliser. Merci.

**Reply by bucanero** [reply]
The official release .pkg file works fine, it has been tested on CFW and there are no issues to execute the app:
https://github.com/bucanero/pkgi-ps3/releases/download/v1.2.0/pkgi-ps3.pkg

if you are having issues, please explain in English your setup, firmware, and what's the problem.

**Reply by AllenWalker1707** [reply]
Hello. This marks this to me. What should I do ? Thank you. 
![20211012_151529](https://user-images.githubusercontent.com/92354541/136964447-02b998b6-d890-4752-830f-561feb45c0f3.jpg)

**Reply by bucanero** [reply]
You need to create your own `pkgi.txt` file with the content and links you own.
The file format is .csv (comma separated), and you can read the instructions here: https://github.com/bucanero/pkgi-ps3#db-formats

**Comment by AllenWalker1707** [comment]
Could you describe step by step what I need to do to put PKGI version 1.2.0 on my ps3? Because I don't understand anything .....

Thank you

Provenance : Courrier<https://go.microsoft.com/fwlink/?LinkId=550986> pour Windows 10

De : Damián ***@***.***>
Envoyé le :mardi 12 octobre 2021 15:51
À : ***@***.***>
Cc : ***@***.***>; ***@***.***>
Objet :Re: [bucanero/pkgi-ps3] PKGi PS3 v1.2.0 (Discussion #80)


You need to create your own pkgi.txt file with the content and links you own.
The file format is .csv (comma separated), and you can read the instructions here: https://github.com/bucanero/pkgi-ps3#db-formats

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/bucanero/pkgi-ps3/discussions/80#discussioncomment-1464981>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AWATP3KNHXUDIHFG2JYFOPDUGQ4MNANCNFSM5FV3FLKQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by AllenWalker1707** [comment]
so ?

Provenance : Courrier<https://go.microsoft.com/fwlink/?LinkId=550986> pour Windows 10

De : Damián ***@***.***>
Envoyé le :mardi 12 octobre 2021 15:51
À : ***@***.***>
Cc : ***@***.***>; ***@***.***>
Objet :Re: [bucanero/pkgi-ps3] PKGi PS3 v1.2.0 (Discussion #80)


You need to create your own pkgi.txt file with the content and links you own.
The file format is .csv (comma separated), and you can read the instructions here: https://github.com/bucanero/pkgi-ps3#db-formats

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/bucanero/pkgi-ps3/discussions/80#discussioncomment-1464981>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AWATP3KNHXUDIHFG2JYFOPDUGQ4MNANCNFSM5FV3FLKQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by Masamune3210** [reply]
So What? People arent here to spoon feed you, especially with that attitude. The definition expected of the file was already told to you to be here https://github.com/bucanero/pkgi-ps3#db-formats
Either make a file yourself or have someone on a forum help you

**Comment by AllenWalker1707** [comment]
Okay……

Provenance : Courrier<https://go.microsoft.com/fwlink/?LinkId=550986> pour Windows 10

De : ***@***.***>
Envoyé le :samedi 16 octobre 2021 10:05
À : ***@***.***>
Cc : ***@***.***>; ***@***.***>
Objet :Re: [bucanero/pkgi-ps3] PKGi PS3 v1.2.0 (Discussion #80)


So What? People arent here to spoon feed you, especially with that attitude. The definition expected of the file was already told to you to be here https://github.com/bucanero/pkgi-ps3#db-formats
Either make a file yourself or have someone on a forum help you

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/bucanero/pkgi-ps3/discussions/80#discussioncomment-1486160>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AWATP3I62QHOECRSTJVTQD3UHEW45ANCNFSM5FV3FLKQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
